DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -485,6 +486,15 @@ class DMVisitorExpression : DMASTVisitor {
             Result = new Expressions.In(expr, container);
         }
 
+        public void VisitInRange(DMASTExpressionInRange expressionInRange) {
+            var value = DMExpression.Create(_dmObject, _proc, expressionInRange.Value, _inferredPath) as Number ?? new Number(0);
+            var startRange = DMExpression.Create(_dmObject, _proc, expressionInRange.StartRange, _inferredPath) as Number ?? new Number(0);
+            var endRange = DMExpression.Create(_dmObject, _proc, expressionInRange.EndRange, _inferredPath) as Number ?? new Number(0);
```

Comment:
This turns anything that isn't a constant `0` into a zero. These can all be variables/dereferences, so it would likely be best to instead create a new `IsInRange` opcode.